packages/client/tests/functional/_utils/setupTestSuiteMatrix.ts

```diff
@@ -59,6 +60,7 @@ function setupTestSuiteMatrix(
   })
 
   for (const { name, suiteConfig, skip } of testPlan) {
+    const generatedFolder = getTestSuiteFolderPath(suiteMeta, suiteConfig)
```

Comment:
Could this be called directly in the test instead?

Reply:
`suteConfig` we have here is not quite the same test suite gets: this one includes the name of the test and all matrix parameters, and we path only matrix parameters to the callback. Unfortunately, name is needed for this function.